dash_bio/component_factory/_clustergram.py

```diff
@@ -136,38 +134,36 @@ def Clustergram(
     imputing should happen along columns, while 'axis=1' indicates
     that it should happen along rows (see: https://scikit
     -learn.org/stable/modules/generated/sklearn.preprocessing.Imputer.html).
-    (Default: None)
 - row_group_marker (list; optional) A list containing the annotations
     for row clusters in the dendrogram. Each annotation is a
     dictionary with the keys 'group_number' (the cluster number to
     highlight), 'annotation' (a string containing the text of the
     annotation), and 'color' (a string representation of the color of
-    the annotation). (Default: None)
+    the annotation).
 - col_group_marker (list; optional): A list containing the annotations for
     column clusters in the dendrogram. Each annotation is a dictionary
     with the keys 'group_number' (the cluster number to highlight),
     'annotation' (a string containing the text of the annotation), and
     'color' (a string representation of the color of the
-    annotation). (Default: None)
+    annotation).
 - tick_font (dict; optional): The font options for ticks, as specified
    in the Plotly graph_objs documentation (see:
-   https://plot.ly/python/reference/#bar-marker-colorbar-tickfont). (Default:
-   None)
+   https://plot.ly/python/reference/#bar-marker-colorbar-tickfont).
 - annotation_font (dict; optional): The font options for annotations,
    as specified in the Plotly graph_objs documentation (see:
-   https://plot.ly/python/reference/#layout-scene-annotations-items-annotation-font). (Default:
-   None)
-- line_width (list/float; optional): The line width for the
+   https://plot.ly/python/reference/#layout-scene-annotations-items-annotation-font).
+- line_width (list | number; default 0.5): The line width for the
     dendrograms. If in list format, the first element corresponds to
```

Comment:
Indentation seems inconsistent... Items above are indented with only one space (within item).

Reply:
Good catch - fixed in 8c5b90d!

Reply:
8c5b90d creates an inconsistency the other way around: items above (lines 150-151 and 153-154) are now indented with three spaces relative to the item (bullet point). You have prepended two spaces instead of one.

Reply:
Fixed in c377939e8b704e56f452a5ad38e9beeaf53256d7 (I fixed the other docstring, too).